### ISSUE TITLE
[BUG] Can't update my WS upload

### ISSUE THREAD

**Issue by OverSavior** [tsunamayo/Starship-EVO#2699]
# Environment
- Starship EVO version: 20w30b

This morning I figured that there was a sticker bugfix update. I went to check my ship and found that it doesn't look the same anymore, so I fixed it adding some more stickers where they were missing. Unfortunately though, when I tried to update the WS of this blueprint there was no such option. So, currently the WS blueprint that I've uploaded still have broken stickers. I'd also like to double one of the earlier posts on the topic of uploading/updating WS blueprints with a custom preview and not auto-generated one.

![image](https://user-images.githubusercontent.com/31092530/88157708-c9c81780-cc13-11ea-815b-2b2a3157f771.png)

**Comment by Kaiser-Indrasil** [comment]
It's been already approved here #2695

**Comment by tsunamayo** [comment]
Yes I guess you spawned a new instance of this blueprint. You can already upload a custom preview, just go to the folder indicated and replace the preview png before uploading. Thanks!
